### PR TITLE
Prac_05_feedback

### PR DESCRIPTION
**Pull request by JiajunJiang-Oscar**

### CODE REVIEW

**Comment by Yikkwokchun** [comment]
I am very sorry that I did not reply to your email in time. I felt a little
unwell today and I slept for a while at night. I just saw the email now.
Thank you for reminding me.


Oscar Jiang ***@***.***>于2024年10月19日 周六22:54写道：

**Comment by JiajunJiang-Oscar** [comment]
Sorry, I didn't find the right pull request and the right name so I emailed you very late, if you can see this, please give me a reply, thanks. @K-lui

**Comment by K-lui** [comment]
Hey Jiajunjiang Oscar, I got your review request.



Kevin Lui | Student
CP1407
Bachelor of Information Technology
Cairns - Nguma-bada Campus

Student ID: 14470127

________________________________

**Review by K-lui — COMMENTED** [review]
This looks good👍

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+hex_code = str(input("Enter the name of the colour to get hex code >>>").title())
+
+while hex_code != "":
+    try:
+        print(f"{hex_code:7} is {HEX_TO_COLOUR[hex_code]}")
+
+    except KeyError:
+        print("Invalid input")
+    hex_code = str(input("Enter short state: ").title())
```
Comment (K-lui) [inline prac_05/hex_colours.py:33]: "Enter short state", I'm don't think this is what the program should be asking for here.

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                "Tiffany": "#0abab5",
+                 "Yellow": "#ffff00",
+                 "White": "ffffff",
+                 "Plum": "#8e4585",
+                 "Peach": "ffe5b4",
+                 "Olive": "#808000",
+                 "Melon": "#febaad"
+                 }
+
+hex_code = str(input("Enter the name of the colour to get hex code >>>").title())
```
Comment (K-lui) [inline prac_05/hex_colours.py:25]: Getting an input without identifying the class is automatically assumed a string, so 'str' may not be needed here.

File: prac_05/word_occurrences.py
Comment (K-lui) [inline prac_05/word_occurrences.py:1]: Good naming

File: prac_05/kivy_layout.kv [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# This is a Kv file.
+# It contains the declarations of the widgets that will be displayed.
+BoxLayout:
+    orientation: 'vertical'
+    Label:
+        text: app.status_text
+        font_size: 60
+        size_hint_y: 0.2
```
Comment (K-lui) [inline prac_05/kivy_layout.kv:8]: One of the instructions are to change something from 20% to 10%. This is yet to be done.

File: prac_05/wimbledon.py
Comment (K-lui) [inline prac_05/wimbledon.py:1]: You have good comments

**Review by K-lui — COMMENTED** [review]
Good stuff. I added a few comments for you to check out👍

**Pull request merged**